features/login.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Logging in, Logging out
+
+  @javascript
+  Scenario: I can log-in and out
+    Given I am logged in as an admin
+    And I am on the Dashboard
+    Then I should see "Howdy"
+
+    Given I am an anonymous user
```

Comment:
I think your test here would be better if you say something like "And then I log out, I should not see .."